group-owner.html

```diff
@@ -182,6 +182,7 @@
 <div class="note">
 <p class="note-title">Note</p>
 <p class="note-text">If a user is not a member of any other group, their name will be in a red box, and they cannot be removed from the group.</p>
+<p class="note-text">Administrator permissions are required to remove a user from their only group.</p>
```

Comment:
Maybe ``Administrator permissions are required to deactivate these users in such a case.``